registry/recipes/packages/py-prompthandler/src/faktion/prompthandler/base.py

```
@@ -0,0 +1,128 @@
+                try:
+                    inner_content = self.get_prompt(  # type: ignore[misc]
+                        name=missing_variable,
+                        context=context,
+                        raise_on_missing_input=raise_on_missing_input,
+                        max_recursion=max_recursion - 1,
+                        _is_recursive=True,
+                        *args,
+                        **kwargs,
+                    )
+                except ValueError as e:
+                    raise e
```

Ruff (0.14.14)

[warning] Remove exception handler; error is immediately re-raised

(TRY203)